DESCRIPTION

```diff
@@ -9,11 +9,14 @@ Description: More about what it does (maybe more than one line)
 License: BSD_2_clause + file LICENSE
 Encoding: UTF-8
 LazyData: true
+Depends:
+    R (>= 2.10)
 Imports:
-    curl,
     V8,
     jsonlite,
-    magrittr
 Suggests: 
-    spelling
+    spelling,
+    magrittr,
```

Comment:
`magrittr` and `usethis` are only in the `./scripts` folder which is not passed on to CRAN.  

These two packages may be removed.

Reply:
should spelling also be removed from 'Suggests'?